PhysicsTools/NanoAOD/python/jetsAK4_Puppi_cff.py

```diff
@@ -149,6 +152,10 @@ def nanoAOD_addDeepInfoAK4(process,addParticleNet,addRobustParTAK4=False,addUnif
         print("Updating process to run UnifiedParTAK4")
         from RecoBTag.ONNXRuntime.pfUnifiedParticleTransformerAK4_cff import _pfUnifiedParticleTransformerAK4JetTagsAll as pfUnifiedParticleTransformerAK4JetTagsAll
         _btagDiscriminators += pfUnifiedParticleTransformerAK4JetTagsAll
+    if addUnifiedParTAK4V1:
```

Comment:
I think it might be easier to reuse `addUnifiedParTAK4` to control the addition of both v1 and v2. Otherwise you need to add the corresponding flag to `nanoAOD_addDeepInfoAK4_switch` too.

Reply:
Will set everything under the same `addUnifiedParTAK4` then